circuit_maintenance_parser/parsers/equinix.py

```diff
@@ -97,7 +99,29 @@ def _parse_table(self, theader_elements, data, impact):  # pylint: disable=no-se
                         continue
                     circuit_info = list(tr_elem.find_all("td"))
                     if circuit_info:
-                        account, _, circuit = circuit_info  # pylint: disable=unused-variable
+                        if len(circuit_info) == 4:
+                            # Equinix Connect notifications contain the IBX name
+                            account, _, _, circuit = circuit_info  # pylint: disable=unused-variable
+                        elif len(circuit_info) == 14:
+                            # Equinix Fabric notifications include a lot of additional detail on seller and subscriber ID's
+                            (
+                                account,
+                                _,
+                                _,
+                                circuit,
+                                _,
+                                _,
+                                _,
+                                _,
+                                _,
+                                _,
+                                _,
+                                _,
+                                _,
+                                _,
+                            ) = circuit_info  # pylint: disable=unused-variable
+                        else:
+                            account, _, circuit = circuit_info  # pylint: disable=unused-variable
```

Comment:
This would protect against exceptions about some circuit_info unexpected structures

```suggestion
                        elif len(circuit_info) == 3:
                            account, _, circuit = circuit_info  # pylint: disable=unused-variable
                        else:
                            return
```